src/views/AdminPanel.vue

```diff
@@ -77,7 +73,7 @@ $headerHeight: 50px;
       text-decoration: none;
 
       &:hover {
-        color:#ccc;
+        color: #ccc;
```

Comment:
may be it is better to have same constants like LIGHT_GRAY  = #ccc and than import it and use in such code

Reply:
It's hard. (discussed)